tools/dataform_assertions/definitions/validation_test/validate_date_row_assertions_unit_test.sqlx

```diff
@@ -0,0 +1,49 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+config {
+    type: "assertion",
+    schema: "dataform_assertions_validation",
+    name: "validate_date_row_assertions_unit_test",
+    tags:[
+      "validate_date_row_assertions_unit_test",
+      "validate_all_unit_tests",
+    ],
+}
+
+######## Run the following steps to validate unit tests ################
+#
+# dataform run --tags date_row_assertions_unit_test;
+# dataform run --tags validate_date_row_assertions_unit_test;
+#
+########################################################################
+
+SELECT 1 AS failure
+FROM(
+  SELECT
+    test_data.input,
+    test_data.expected_output,
```

Comment:
```suggestion
SELECT 'Unit Test Failed' AS test_result, input, expected_output
FROM(
  SELECT
    test_data.input,
    test_data.expected_output,
```

Reply:
Make this change for all validation sqlx. It returns more helpful information to pin point the failed test cases